src/inc/corinfo.h

```diff
@@ -1312,6 +1312,7 @@ enum CORINFO_RUNTIME_LOOKUP_KIND
 struct CORINFO_LOOKUP_KIND
 {
     bool                        needsRuntimeLookup;
+    WORD                        runtimeLookupFlags;
```

Comment:
What is the range of values for this field? Is it ever zero if `needsRuntimeLookup` is true?